truss/cli/cli.py

```diff
@@ -419,7 +435,7 @@ def push(
     if service.is_draft:
         draft_model_text = """
 |---------------------------------------------------------------------------------------|
-| Your model has been deployed as a draft. Draft models allow you to                    |
+| Your model has been deployed as a development model. Development models allow you to  |
 | iterate quickly during the deployment process.                                        |
 |                                                                                       |
 | When you are ready to publish your deployed model as a new version,                   |
```

Comment:
This still says "version", I suggest something like `When you are ready to publish a new deployment of your model, pass`...

Reply:
good catch, thanks!